lib/dslash_gamma_helper.cu

```diff
@@ -103,4 +103,44 @@ namespace quda {
   // Applies a gamma5 matrix to a spinor (wrapper to ApplyGamma)
   void gamma5(ColorSpinorField &out, const ColorSpinorField &in) { ApplyGamma(out,in,4); }
 
+  template <typename Float, int nColor> class TauApply : public TunableKernel2D {
+    ColorSpinorField &out;
+    const ColorSpinorField &in;
+    const int d;
+    unsigned int minThreads() const { return in.VolumeCB(); }
+
+  public:
+    TauApply(ColorSpinorField &out, const ColorSpinorField &in, int d) :
+      TunableKernel2D(in, in.SiteSubset()),
+      out(out),
+      in(in),
+      d(d)
+    {
+      apply(device::get_default_stream());
+    }
+
+    void apply(const qudaStream_t &stream) {
+      TuneParam tp = tuneLaunch(*this, getTuning(), getVerbosity());
+      launch<Tau>(tp, stream, GammaArg<Float, nColor>(out, in, d));
+    }
+
+    void preTune() { out.backup(); }
+    void postTune() { out.restore(); }
+    long long flops() const { return 0; }
```

Comment:
This function can be deleted since the parent default return 0;

Reply:
https://github.com/lattice/quda/pull/1452/commits/378e5c24afabe99142a9fd48102492edd57d882c